configuration/analyzer_config.json

```diff
@@ -355,6 +355,18 @@
     "python_module": "ha_get.HybridAnalysisGet",
     "soft_time_limit": 30
   },
+  "IPInfo": {
```

Comment:
I think something went wrong with a local merge. I suggest you to start again from a clean `develop` branch and add your analyzer on top of it